ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -720,6 +768,10 @@ case "$CEPH_DAEMON" in
       OSD_TYPE="directory"
       start_osd
       ;;
+   osd_directory_simple)
```

Comment:
Perhaps name this `osd_directory_single` instead of simple?  It is simpler, yes, but from the user's perspective, it is "single."